src/components/AddComment.jsx

```diff
@@ -0,0 +1,85 @@
+import FloatingLabel from 'react-bootstrap/FloatingLabel';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import { postComment } from './utils';
+import { useState } from 'react';
+import { UserContext } from './UserContext';
+import { useContext } from 'react';
+
+export const AddComment = ({ setComments, id, setShowToast, setToastMsg }) => {
+	const { user } = useContext(UserContext);
+	const [newComment, setNewComment] = useState();
+	const [isLoading, setIsLoading] = useState();
+	const [btnMsg, setBtnMsg] = useState('Post');
+
+	const showError = (msg) => {
+		setToastMsg('Your comment was not posted! Pleas try again.');
+		setShowToast(true);
+	};
+
+	const handleSubmit = (e) => {
+		e.preventDefault();
+		const commentBeingPosted = newComment;
```

Comment:
Not a must, but it can be useful to have a minimum length value for the comment that is posted. Low effort comments like a single letter can be avoided! Just some additional feedback, this must NOT be implemented